src/Solvers/Greedy.jl

```diff
@@ -2,7 +2,8 @@ using DataStructures: BinaryMinHeap
 using Base: @kwdef
 using OptimizedEinsum: removedsize, ssa_to_linear, nonunique, ContractionPath, popvalue!
 using Combinatorics: combinations
-
+using Random: seed!
+using Distributions: Categorical
```

Comment:
Why do we need this `Categorical` from the `Distributions` package?

Reply:
This is to chose randomly from a set which is weighted by some probabilities. We do this in line 216,
```julia
    (chosen, ) = range(1,n)[rand(Categorical(energies))]
```

Reply:
Okay, but the `range` is not needed.